docs/google/devices.md

```diff
@@ -105,6 +144,16 @@ of the circuit by pushing them back to the next non-commuting operator. If the r
 circuit still contains Z operations, they should be aggregated into their own moment,
 if possible.
 
+### Use caution with symbols
+
+Symbols are extremely useful for constructing parameterized circuits (see above).  However,
+only some sympy formulas can be serialized for network transport to the engine.
+Currently, only linear combinations of symbols are supported.  See `cirq.google.arg_func_langs`
```

Comment:
We support sums and products, not just linear combinations. For example, `x*x*y` is allowed.

Reply:
Changed.